### PR TITLE
v1

### PR DESCRIPTION
**Pull request by Stanko**
* Updated dependencies.
* Switched to redux-saga.
* Switched to postcss.
* Async server data.
* Removed unused files.
* Refactoring.
* Updated documentation.

### CODE REVIEW

**Comment by Stanko** [comment]
@oliverdore-work @mpavlicevic 

Guys please take a look, I made a bunch of changes. Documentation still needs polishing.

Check updated readme [here](https://github.com/workco/marvin/tree/v1).

cc @radospavlicevic @novojin

File: source/js/sagas/people.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import {
+  GET_PEOPLE_START,
+  GET_PEOPLE_ERROR,
+  GET_PEOPLE_SUCCESS,
+} from 'actions/people';
+import api from 'api';
+
+// -------- Get people
+
+function createGetPeope(isServer = false) {
```
Comment (oliverdore-work) [inline source/js/sagas/people.js]: @Stanko Typo here, `createGetPeople`

File: source/js/server.js [hunk 2]
```diff
@@ -47,13 +54,55 @@ app.use((req, res) => {
     res.status(context.status || 200).send(serverHtml);
   }
 
-  // TODO how to handle 50x errors?
+  // TODO handle 50x errors
```
Comment (oliverdore-work) [inline source/js/server.js]: @Stanko 😆

File: source/css/base/vars.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  --font-family-serif: Garamond, "Times New Roman", serif;
+  --font-family-sans-serif: "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+  --grey-border: #eee;
+  --grey-text: #aaa;
+  --black: #000;
+  --white: #fff;
+}
+
+@custom-media --sm (max-width: 500px);
```
[the remark this answers is not in the document]
Comment (Stanko) [reply source/css/base/vars.css]: Hehe personally I prefer it without `bp`, but we can add it. This CSS was intended just as an example, 99% of people are going to override it all together.

File: source/js/server.js (hunk 2 above)
Comment (Stanko) [reply source/js/server.js]: TBH I don't know how atm 🙃 I need to do some more reading on express.

**Pull request merged**